firebase-functions/functions/src/firebase/calculateResult.ts

```diff
@@ -14,7 +14,16 @@ export const calculateResult = functions.firestore.document('/players/{documentI
     try {
       const roomId = await getCurrentRoomId(context.params.documentId as string);
       const initiallyActivePlayers = await getActivePlayersInRoom(roomId);
-      if (!hasEveryoneChosen(initiallyActivePlayers)) {
+      const numberOfPlayersInTheRoom = await getNumberOfPlayers(roomId);
+      const isThereOnlyOnePlayerInTheRoom = numberOfPlayersInTheRoom < 2;
+      if (isThereOnlyOnePlayerInTheRoom)
+      if (isThereOnlyOnePlayerInTheRoom && !hasEveryoneChosen(initiallyActivePlayers)) {
+        return;
+      }
+      const isThereAForfeitPlayer = numberOfPlayersInTheRoom >  1 && initiallyActivePlayers.length === 1;
```

Comment:
Extra whitespace, check the linter pliiis

Reply:
I diiiiiiiiid!